docs/protocol/data-requests/rng.md

```diff
@@ -0,0 +1,52 @@
+# Random Number Generation
+
+## Witnet RNG request
+The Witnet RNG request shares the same structure as a normal request with the 3 stages (retrieval, aggregation and 
```

Comment:
What do you think about use a more specific word such as `HTTP-GET` or even `standard`? 
```suggestion
The Witnet RNG request shares the same structure as an HTTP-GET request with the 3 stages (retrieval, aggregation and 
```